src/next/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,333 @@
+import {
+  type HeroGradientProps,
+  type HeroSplitProps,
+  type HeroFloatingProps,
+  type HeroCopyLedProps,
+  type HeroFloatingImageProps,
+} from "~/common/Hero"
+import Button from "~/next/components/Button"
+
+export type HeroProps =
+  | HeroGradientProps
+  | HeroSplitProps
+  | HeroFloatingProps
+  | HeroCopyLedProps
+  | HeroFloatingImageProps
+
+const HeroGradient = ({
+  alignment = "left",
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+}: HeroGradientProps) => {
+  return (
+    <section
+      className="bg-cover bg-center bg-no-repeat h-[32rem]"
+      style={{
+        backgroundImage: `url('${backgroundUrl}')`,
+      }}
+    >
+      <div
+        className={`h-full w-full ${
+          alignment === "left" ? "bg-gradient-to-r" : "bg-gradient-to-l"
+        } from-[rgba(0,0,0,85%)] md:from-[rgba(0,0,0,75%)] xl:from-[rgba(0,0,0,65%)]`}
```

Comment:
the from color seems harcoded here?

Reply:
This is just the opacity. Users can't select a different background colour for this gradient underlay.